site/_posts/2023-07-11-get-more-value-out-of-fusionauth.md

```diff
@@ -0,0 +1,321 @@
+author: Mark Robustelli
+image: blogs/get-more-value/get-more-value-header.png
+category: article
+tags: api user maintenance windows forms c#
+excerpt_separator: "<!--more-->"
+---
+
+So, you have just set up FusionAuth and are excited to take your new [customer identity and access management](/articles/ciam/) platform out for a spin. You spend some time setting up your Users, Applications, Tenants, etc. in the robust front end provided. Things are looking great. You have integrated your applications and are humming along. Now comes a real test, user maintenance.
+
+<!--more-->
+
+<br>
+
+## Introduction
+*“A new user with extra pickles, please.”*
+
+Customizing the order for your favorite food can make a good meal better. Customizing your end-users' experience with user maintenance can make their experience with your application even better.
+
+For any [secure application](/articles/security/steps-secure-your-authentication-system), user maintenance is a must. As developers, we can often overlook the time and effort it takes to maintain the users access and permissions for the shiny new application we created. In an ideal world, we do not want to be the ones to maintain the users. We want to give the task of user maintenance to the power users of the applications by making them application administrators. While those power users may be great at using the application, you don’t necessarily want to make them expert users of the FusionAuth front end. Well, we at FusionAuth would think it would be great if everyone was a master with our front end, but you may not share that same vision.
+
+We can harness the full power of FusionAuth in our application, hand off the user maintenance to the power users, and make it seamless for them. The application will need to ensure the user enters the correct information (**Validation**), guide them through setting up the new user correctly (**Workflow**), and we want to make the process feel like it is part of our application (**Branding**). We can accomplish all these through the use of the FusionAuth APIs.
+
+The examples below are written with the assumption that you are familiar with how to make REST calls using the [HttpClient](https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient?view=net-7.0) in C#, the use of [asynchronous](https://learn.microsoft.com/en-us/dotnet/csharp/asynchronous-programming/) calls and Windows Forms applications.  In order for the sample application to run you must have a [working FusionAuth server](/docs/v1/tech/5-minute-setup-guide). Even if you are not totally familiar with these concepts, the snippets should be easy enough to follow with general programming knowledge. If you need a refresher on some of these topics, there are some links at the end.
+
+<br>
```

Comment:
Why do you have this?

Reply:
I wanted to give some context to the sample code and allow the newer developers a chance to brush up on the technology if they were not familiar with it.  I can see your point though and will remove.

Reply:
It's okay to have it, I was asking. I think explaining why you mention it would give the context.

Reply:
I meant the <br> tag, sorry for being imprecise.

Reply:
updated